lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -164,7 +164,11 @@ def add_copy_resources_script_phase
             phase = create_or_update_build_phase(native_target, phase_name)
             script_path = target.copy_resources_script_relative_path
             phase.shell_script = %("#{script_path}"\n)
-            # TODO: Add input and output paths for the copy resources script phase.
+            resource_paths_by_config = target.resource_paths_by_config
+            unless resource_paths_by_config.empty?
+              phase.input_paths = [target.copy_resources_script_relative_path, *resource_paths_by_config.values.flatten.uniq]
+              phase.output_paths = ['${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}']
```

Comment:
This one is for the output but the idea is that giving a folder its contents will be monitored by Xcode.

If any file changes within this folder then this will re-trigger the task.

The "lazy" part is that I am not explicitly defining the list of resources to monitor and instead I use the entire folder. This means that if other files change in there (outside of the ones in the scripts concerns) it will re-trigger the script.

Based on executing this locally it has been fine and the script is avoided. In our flagship a target takes 8 seconds to run this script and with this change its completely avoided!